docs: Favor the Admin app over the CLI to setup FastStore (#1454)
vtex/faststore: apps/docs/docs/how-to-guides/platform-integration/vtex/setting-up-an-account.md

## Step by step

> In this step by step, you'll install the FastStore Admin app in your account via the VTEX IO CLI.

Reviewer: ```suggestion
In this step by step, you will install the FastStore Admin app in your account via the VTEX IO CLI.
```

> 1. Install the FastStore Admin app in your account:

Reviewer 2: Please add a first step `Log in to your VTEX account` before this one

   ```
   vtex install vtex.admin-faststore
   ```

> 2. Open the Admin app at `https://{workspace}--{account}.myvtex.com/admin/faststore` and follow the setup steps.

Reviewer 2: Are we recommending using a workspace (other than master) on this setup?
I only worry that people start to think that they'll need to use this specific workspace when using faststore

> Once the setup completes, all dependencies needed for developing a FastStore website will have been created: `Happy coding on FastStore 🎉`.

Author: ```suggestion
Once the setup completes, all dependencies needed for developing a FastStore website will have been installed: `Happy coding on FastStore 🎉`.
```

> Once the setup completes, all dependencies needed for developing a FastStore website will have been installed: `Happy coding on FastStore 🎉`.

Reviewer: ```suggestion
Once the setup completes, all dependencies needed for developing a FastStore website will have been installed.
```
